zerver/webhooks/github/githubsponsors.md

```diff
@@ -15,8 +17,22 @@ Get GitHub Sponsors notifications in Zulip!
 1. Set **Payload URL** to the URL constructed above. Set **Content type**
```

Comment:
constructed -> generated

Reply:
And can you add a comma before any case of "and" in these instructions as well. There's one below this line and in the prior instruction.